index.bs

```diff
@@ -2155,17 +2162,13 @@ when invoked, must run these steps:
     2. [=Queue a task=] to run these substeps:
 
         1. If |result| is an error, set the [=request/error=]
-            of |request| to |result| and dispatch an event at
-            |request|. The event must use the [=Event=] interface
-            and set the {{Event/type}} attribute to
-            "<code>error</code>". The event does bubble but is not
-            cancelable.
+            of |request| to |result| and [=fire an event=] named
+            <code>error</code> at |request| with its
+            {{Event/bubbles}} attribute initialized to true.
 
         2. Otherwise, set the [=request/result=] of |request|
-            to |result| and dispatch an event at |request|. The event
-            must use the [=Event=] interface and set the
-            {{Event/type}} attribute to "<code>success</code>". The
-            event does not bubble and is not cancelable. If the steps
+            to |result| and [=fire an event=] named
```

Comment:
Thank you for clarifying! The GitHub edit window is too small... I'll review the changes with the HTML on the right.